utilities/merge_operators/cassandra/merge_operator.cc

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2017-present, Facebook, Inc.  All rights reserved.
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree. An additional grant
+// of patent rights can be found in the PATENTS file in the same directory.
+
+#include "merge_operator.h"
+
+#include <memory>
+#include <assert.h>
+
+#include "format.h"
```

Comment:
Seems like this should be utilities/merge_operators/cassandra/format.h following the convention in other files.